pipelines/HSC/ProcessCcdCalibrate.yaml

```diff
@@ -2,7 +2,7 @@ description: A set of tasks to run when processing raw images, specialized for H
 
 instrument: lsst.obs.subaru.HyperSuprimeCam
 imports:
-  - location: $AP_PIPE_DIR/pipelines/_ingredients/ProcessCcd.yaml
+  - location: $AP_PIPE_DIR/pipelines/_ingredients/ProcessCcdCalibrate.yaml
```

Comment:
Did you make sure this doesn't break the `ApPipe` or `ApPipeCalibrate` pipelines?

Reply:
It shouldn't have affected ApPipe at all, and I checked the other.

Reply:
Please check to make sure that nothing that shouldn't have happened, happened.

Reply:
Yes, `ApVerify` should be enough.